index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href ="index.css">
+</head>
+<body>
+
+
+    <form id ="myform">
+<div class="main">
+    <div id ="header">Registration form</div>
+
+<div class="input-container">
+    <label for="login">Login ID</label>
+    <input id="l" class="heads" name="login" type="text">
+</div>
+
+<div class="input-container">
+    <label for="Email">Email</label>
+    <input id="E" class="heads" name="email" type="text">
+</div>
+
+<div class="input-container">
+    <label for="Name">Name</label>
+    <input id="N" class="heads" name="name" type="text">
```

Comment:
for attribute of label tag and id attribute for input tag should always same.
Please study more about for and id attributes